### PR TITLE
(Inventory Branch) Fix End Gateway causing crashes

### LINKED ISSUE #283: Loading in with certain blocks causes crash (Inventory branch)
**Describe the bug**

On the inventory branch, going into the End crashes the Bedrock client. Geyser and the Java server still believe the user is connected until they time out. The master branch has no such issue.

**To Reproduce**

**Server Version**
Paper 161

**Geyser Version**
Geyser Inventory 9

**Minecraft: Bedrock Edition Version**
1.14.30

### PR DESCRIPTION
**Pull request by DoctorMacc**
Fixes #283 .

Xbox still crashes upon loading The End when joining a game - this will not be addressed with this PR.

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.geysermc.connector.network.translators.block.entity;
```
Comment (RednedEpic) [inline connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java:1]: Missing license header.

File: connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
+    public List<Tag<?>> translateTag(CompoundTag tag) {
+        List<Tag<?>> tags = new ArrayList<>();
+        tags.add(new IntTag("Age", (int) (long) tag.get("Age").getValue()));
+        // Java sometimes does not provide this tag, but Bedrock crashes if it doesn't exist
+        // Linked coordinates
+        List<IntTag> tagsList = new ArrayList<>();
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "X")));
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "Y")));
+        tagsList.add(new IntTag("", getExitPortalCoordinate(tag, "Z")));
```
Comment (RednedEpic) [inline connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java:23]: Is this correct? Shouldn't this be lowercase?

Comment (DoctorMacc) [reply connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java:23]: Yes; The ExitPortal CompoundTag in Java has the axis letters capitalized.

Comment (RednedEpic) [reply connector/src/main/java/org/geysermc/connector/network/translators/block/entity/EndGatewayBlockEntityTranslator.java:23]: :microjang:

**Pull request merged**